tests/PerimeterxCookieValidatorTest.php

```diff
@@ -275,6 +275,38 @@ public function testCookieHmacValid() {
         );
     }
 
+    public function testCookieVerificationFailedOnSensetiveRoute() {
+
+        // far future, consistent cookie, 2116-11-14T00:00:00Z
+        $cookie_time = '4634841600000';
+        $cookie_uuid = self::COOKIE_UUID;
+        $cookie_vid = self::COOKIE_VID;
+        // calculated at time of writing
+        $cookie_hmac = '826bb9324795dd2e621e133372a627f7b5d9523978fd1c3337389b9fa1f5cbc7';
+        $cookie_score_a = 0;
+        $cookie_score_b = 0;
+
+        $pxCookie = $this->encodeCookie(
+                $this->createCookie($cookie_time, $cookie_vid, $cookie_uuid, $cookie_hmac, $cookie_score_a, $cookie_score_b)
+            );
+        $userAgent = self::USER_AGENT;
+        $ip = self::IP;
+        $pxCtx = $this->getPxContext($pxCookie, $userAgent, $ip, true);
```

Comment:
don't get this one - we initialize the context with the desired state without testing how configure a path cause risk request to be considered sensitive